core/src/main/scala/org/apache/spark/TaskContext.scala

```diff
@@ -182,7 +182,6 @@ abstract class TaskContext extends Serializable {
    * about the resource. Please refer to [[org.apache.spark.resource.ResourceInformation]] for
    * specifics.
    */
-  @Evolving
```

Comment:
this was just added in 3.0, it should be left.

Reply:
Oops right, I see it's part of GPU resources